app-drawer/app-drawer.html

```diff
@@ -22,12 +48,20 @@
       left: 0;
       width: 256px;
       overflow: hidden;
-      background-color: var(--x-drawer-background-color, #fff);
+      background-color: var(--app-drawer-background-color, white);
       visibility: hidden;
+
       -webkit-transform: translate3d(-100%, 0, 0);
       transform: translate3d(-100%, 0, 0);
-      -webkit-transition: -webkit-transform 0.4s cubic-bezier(0.42, 0, 0.33, 1.01);
-      transition: transform 0.4s cubic-bezier(0.42, 0, 0.33, 1.01);
+
+      -webkit-transition-property: -webkit-transform;
+      transition-property: transform;
```

Comment:
Can we just do transition on `visibility` so it can automatically handle the visibility toggling with the right timing?  This way we can remove all the js code like `_hasOpenedChanged` and `_onTransitionEnd`